x-pack/legacy/plugins/apm/public/components/shared/charts/CustomPlot/plotUtils.js

```diff
@@ -75,22 +78,26 @@ export function getPlotValues(
     y: yScale,
     yTickValues,
     XY_MARGIN,
-    XY_HEIGHT,
-    XY_WIDTH: width
+    XY_HEIGHT: height || XY_HEIGHT,
+    XY_WIDTH: width,
+    ...(stacked ? { stackBy: 'y' } : {})
```

Comment:
Does this need to be conditional? Doesn't it only take effect when things are stacked? So setting it to `stackBy: 'y'` always only affects `stacked` charts.